### PR TITLE
Tron

### PR DESCRIPTION
**Pull request by OlegGordiichuk**

### CODE REVIEW

File: Sources/Tron/TronSighn.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Copyright © 2017-2018 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import UIKit
+
+public struct TronSighn {
```
Comment (vikmeup) [inline Sources/Tron/TronSighn.swift]: TronSighn.swift => `TronSign.swift` ? same with a class name

File: .swiftlint.yml [hunk 2]
```diff
@@ -20,6 +20,10 @@ opt_in_rules:
 included:
   - Sources
   - Tests
+excluded: 
+   - Sources/Tron/Protobuf/Tron.pb.swift
+   - Sources/Tron/Protobuf/Discover.pb.swift
+   - Sources/Tron/Protobuf/Contract.pb.swift
```
Comment (alejandro-isaza) [inline .swiftlint.yml]: We can exclude the folder with `Sources/Tron/Protobuf`

Comment (OlegGordiichuk) [reply .swiftlint.yml]: done

**Comment by vikmeup** [comment]
@OlegGordiichuk if this works, let's merge it.

**Pull request merged**